src/main/python/community-detector/fixtures/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.14.2
```

Comment:
why do you need to specify this deps?
sourced-ml already has them: https://github.com/src-d/ml/blob/master/requirements.txt